third_party/rust/cxx/v1/crate/Cargo.toml.orig

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "cxx"
```

Comment:
Is it possible to handle this as a submodule or `DEPS` entry rather than embedding the entire source tree directly into our git repo?

Reply:
This was copied from chromium where it is also checked out inside the repo, with patches applied to it.

Reply:
Fixed in 1961e09dad30d0560f2f1e3208643279d2ed3fe2

Reply:
The fix above had to be reversed as the version checked out in the tree comes from Chromium, and it has patches applied to it that we need.

Reply:
I see. It'd be good to document this somewhere at least, so that
a) it's easier to identify how to keep this up to date when the Chromium version changes
b) we can put it in DEPS and remove it from the source tree when Chromium's changes will (presumably) be upstreamed

Reply:
I would advise doing this separately as a PR. Checking out this dependency in tree is related to an issue linking rust code on mac, which prevents upgrading the toolset to rustc 1.60, so some better understanding of how to move forward has to be reached. This has been added for now to unblock cr109.